### ISSUE TITLE
Tendermint crashes with Wrong Block.Header.LastResultsHash error 

### ISSUE THREAD

**Issue by cloudwheels** [dashevo/js-drive#391]
## Expected Behavior
Tendermint server syncs blocks across the network

## Current Behavior
The following error occurs on a new tendermint install (via mn-bootstrap 0.15.1 full node setup)
```
[2020-10-02|10:46:02.875] Committed state                              module=state height=97 txs=1 appHash=
panic: Failed to process committed block (98:AB707244671FBED72939926ACE00E4B5DDF1D3024F88DD69766672FE567BDDD2): Wrong Block.Header.LastResultsHash.  Expected 6E340B9CFFB37A989CA544E6BB780A2C78901D3FB33738768511A30617AFA01D, got 697AC0DA637A4D63975EEEC4D0114CA918111C81B7B2C0DD7F9BE63F9EC40B40

goroutine 255 [running]:
github.com/tendermint/tendermint/blockchain/v0.(*BlockchainReactor).poolRoutine(0xc000088000)
    /home/archie/Documents/tendermint/blockchain/v0/reactor.go:344 +0x12ff
created by github.com/tendermint/tendermint/blockchain/v0.(*BlockchainReactor).OnStart
    /home/archie/Documents/tendermint/blockchain/v0/reactor.go:118 +0x84
I[2020-10-02|10:46:04.756] Found private validator                      module=main keyFile=/tendermint/data/priv_validator_key.json stateFile=/tendermint/data/priv_validator_state.json
I[2020-10-02|10:46:04.757] Found node key                               module=main path=/tendermint/data/node_key.json
I[2020-10-02|10:46:04.757] Found genesis file                           module=main path=/tendermint/config/genesis.json
ERROR: Failed to create node: error during handshake: error on replay: Wrong Block.Header.LastResultsHash.  Expected 6E340B9CFFB37A989CA544E6BB780A2C78901D3FB33738768511A30617AFA01D, got 697AC0DA637A4D63975EEEC4D0114CA918111C81B7B2C0DD7F9BE63F9EC40B40
```

The same problem / error has been occuring sporadically over a number of previous build (since April 2020) 

## Possible Solution

It has been suggested by @shumkov that this is due to a source of non-determinism during block processing:
(https://discordapp.com/channels/670271785974890526/681541036929318994/716330094607597609 https://discordapp.com/channels/670271785974890526/681541036929318994/728642336480821300 https://discordapp.com/channels/670271785974890526/670540408593448980/737998356869283911)

and that further monitoring / logging is needed (https://discordapp.com/channels/670271785974890526/681541036929318994/717838241792786532)

## Steps to Reproduce (for bugs)
run mn start -f using mn-bootstrap package 15.0.1 and monitor tendermint service

## Context
Prevents use of L2 block explorer

## Your Environment
mn-bootstrap on ubuntu (all other services running correctly)

**Comment by cloudwheels** [comment]
I understand that the same issue has also been experienced by @strophy

**Comment by shumkov** [comment]
Debug logging is already implemented in Drive. Pl￼ease enable it passing additional env to Drive service `LOGGING_LEVEL=debug` and provide logs, then we will be able to compare it with logs on evonet (hope debug logging is enabled there).

**Comment by cloudwheels** [comment]
Maybe we can incorporate setting log levels into the mn-bootstraps menu??

**Comment by shumkov** [comment]
Yeah, make sense.

**Comment by shumkov** [comment]
Must be solved with 0.16 version
